kotlin-sqlobject/pom.xml

```diff
@@ -0,0 +1,137 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+~   Licensed under the Apache License, Version 2.0 (the "License");
+~   you may not use this file except in compliance with the License.
+~   You may obtain a copy of the License at
+~
+~   http://www.apache.org/licenses/LICENSE-2.0
+~
+~   Unless required by applicable law or agreed to in writing, software
+~   distributed under the License is distributed on an "AS IS" BASIS,
+~   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+~   See the License for the specific language governing permissions and
+~   limitations under the License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.jdbi</groupId>
+        <artifactId>jdbi3-parent</artifactId>
+        <version>3.0.0-beta1-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>jdbi3-kotlin-sqlobject</artifactId>
+    <name>jdbi SqlObject Kotlin plugin</name>
+
+    <developers>
+        <developer>
+            <name>Matthew Hall</name>
+            <url>https://github.com/qualidafial</url>
+            <roles>
+                <role>Project maintainer</role>
+            </roles>
+        </developer>
+        <developer>
+            <name>Jayson Minard</name>
+            <url>https://github.com/apatrida</url>
+            <roles>
+                <role>Original project creator</role>
+            </roles>
+        </developer>
+    </developers>
+
+    <dependencies>
+
+        <dependency>
+            <groupId>org.jetbrains.kotlin</groupId>
+            <artifactId>kotlin-runtime</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>org.jetbrains.kotlin</groupId>
+            <artifactId>kotlin-stdlib</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>org.jetbrains.kotlin</groupId>
+            <artifactId>kotlin-reflect</artifactId>
+        </dependency>
+
+        <dependency>
```

Comment:
Indeed! I also think that kotlin-runtime dependency should not be explicitly listed. However if I leave it out the build fails:

```
[INFO] --- maven-dependency-plugin:2.10:analyze-only (default) @ jdbi3-kotlin-sqlobject ---
[WARNING] Used undeclared dependencies found:
[WARNING]    org.jetbrains.kotlin:kotlin-test:jar:1.0.6:test

[ERROR] Failed to execute goal org.apache.maven.plugins:maven-dependency-plugin:2.10:analyze-only (default) on project jdbi3-kotlin-sqlobject: Dependency problems found -> [Help 1]
```
Any idea how to fix it? I wonder if dependency check can be disabled for these modules?